commands/zdiffstore.md

```diff
@@ -1,7 +1,6 @@
-Computes the difference between the first set and all the successive sorted
-sets given by the specified keys, and stores the result in `destination`.
-It is mandatory to provide the number of input keys (`numkeys`) before passing
-the input keys.
```

Comment:
i think the previous text was clearer.
why did you change it?
i.e. what did you want to improve? maybe we can find another way to have both?

what i liked is that the first sentence was clear about that this command does, without getting into the fine (confusing) details.

i.e:
```
Computes the difference between the first set and all the successive sorted
sets given by the specified keys, and stores the result in `destination`.
```

maybe you were bothered by the fact that the "first" one is actually the destination?
so maybe something like:
```
Computes the difference between the first input set and all the successive input
sets given by the specified keys, and stores the result in `destination`.
```

Reply:
I adapted the text to follow the "convention" set by previous commands. I also didn't like the "It is mandatory..." part so that was another trigger for the change.

Reply:
@itamarhaber so the current:
```
Computes the difference of `numkeys` sorted sets given by the specified keys,
between the first and all the successive sets, and stores the result in
`destination`.
```
seems clear to you? (to me it looks confusing)